cypress/integration/pages/articles/tests.js

```diff
@@ -168,9 +168,8 @@ export const testsThatFollowSmokeTestConfig = ({
               );
 
               if (captionBlock) {
-                const {
-                  text,
-                } = captionBlock.model.blocks[0].model.blocks[0].model;
+                const { text } =
```

Comment:
This particular styling change is making my code editor want to switch back to the old format, along with all the other similar blocks of code. And I can't this particular linting change in the prettier 2.3.0 [changelogs](https://prettier.io/blog/2021/05/09/2.3.0.html#javascript-1)

```
const {
   text,
} = captionBlock.model.blocks[0].model.blocks[0].model;
```

Reply:
I think it relates to the [Format assignments more consistently](https://prettier.io/blog/2021/05/09/2.3.0.html#format-assignments-more-consistently-10222httpsgithubcomprettierprettierpull10222-10643httpsgithubcomprettierprettierpull10643-10672httpsgithubcomprettierprettierpull10672-by-thorn0httpsgithubcomthorn0-10158httpsgithubcomprettierprettierpull10158-by-sosukesuzukihttpsgithubcomsosukesuzuki) section. My code editor seems to accept it though, it doesn't try to auto-format it to an older format on my end. The new format appears to be controlled by the `printWidth` config value, which defaults to `80`. Increasing it to `120` brings it back to more the style we are accustomed to.

Tbh I'm not sure I really like the changes as there are some areas that seem less readable to me. More than likely habit of having it in that format for so long.